students/ssankura/lesson04/mailroom.py

```diff
@@ -0,0 +1,156 @@
+'''
+Lesson04 - Assignment 01 - Mailrom.py
+Implementation of the Mailroom functionalities 
+- 1 - Send a Thank You
+- 2 - Create a Report
+- 3 - Send letters to everyone
+- 4 - Quit
+'''
+
+#!/usr/bin/env python3
+
+
+''' Global donor data dictionary'''
+donor_data = {}
+
+'''
+Populate data in the global - donor_data dictionary
+'''
+def populate_data():
+	global donor_data
```

Comment:
you don't need this since you already defined at the top of the file outside of the functions